app/forms/personal_key_form.rb

```diff
@@ -32,4 +33,11 @@ def extra_analytics_attributes
   def reset_sensitive_fields
     self.personal_key = nil
   end
+
+  def send_personal_key_sign_in_notification
+    UserMailer.personal_key_sign_in(user.email_address.email).deliver_now
+    MfaContext.new(user).phone_configurations.map do |phone_configuration|
```

Comment:
What is the reason for using map instead of each?

Reply:
I think that's there because I copy/pasted. I can change it up.